cinch/playbooks/install-rhel7.yml

```diff
@@ -76,25 +78,30 @@
       args:
         creates: "{{ venv_dir }}/lib/python2.7/site-packages/setuptools"
 
-    - name: install cinch using pip
+    - name: install released versions of cinch+linch-pin using pip
       pip:
         name: cinch
         virtualenv: "{{ venv_dir }}"
         extra_args: -U
+      when: not (latest_tip|bool)
 
     # This pip install should be non-editable, but the pip module in Ansible
     # 1.8.4. does not support that flag
-    - name: install linch-pin from master branch instead of latest release from pypi
+    - name: install latest tip of cinch+linch-pin instead of latest release from pypi
       command: >-
-        "{{ python }}" "{{ venv_dir }}/bin/pip install" -U
-        git+https://github.com/CentOS-PaaS-SIG/linch-pin.git@master
-      when: (linchpin_master|bool)
+        "{{ python }}" "{{ venv_dir }}/bin/pip" install -U
```

Comment:
Inclusion of the {{ python }} argument here is superfluous. By directly calling the particular executable out of $VENV/bin/, it automatically is keyed through its she-bang line to use the virtual environment's Python file. No harm here, just unnecessary code.